jenkinsfile-examples/sonarqube/Jenkinsfile

```diff
@@ -7,4 +7,13 @@ node {
     withSonarQubeEnv {
         sh "./gradlew clean sonarqube"
     }
+    
+    // later...when you want to continue, 
+    // but only after the quality gate has passed
+    timeout(time: 1, unit: 'HOURS') {
+        def qg = waitForQualityGate()
+        if (qg.status != 'OK') {
+            error "Pipeline aborted dur to quality gate failure: ${qg.status}"  
```

Comment:
There is a small typo at "dur";

error "Pipeline aborted due to quality gate failure: ${qg.status}"